src/components/Gradebook/footer.scss

```diff
@@ -0,0 +1,168 @@
+.max-width-222 {
+  max-width: 222px;
+}
+
+.max-width-264 {
+  max-width: 264px;
+}
+
+.max-width-1180 {
+  max-width: 1180px;
+}
+
+.max-height-39 {
+  max-height: 39px;
+}
+
+.d-grid {
+  display: grid;
+}
+
+$border-1: 1px solid #e7e7e7;
+
+footer {
+  background-color: #fcfcfc;
+
+  ul {
+    padding: 0px;
+    margin: 0px;
+  }
+
+  .area-1 {
+    grid-column: 1;
+    grid-row: 1;
+    border-bottom: $border-1;
+    padding-bottom: 1rem;
+  }
+
+  .area-2 {
+    grid-column: 1;
+    grid-row: 2;
+    border-bottom: $border-1;
+    padding: 1rem 0;
+  }
+
+  .area-3 {
+    grid-column: 1;
+    grid-row: 3;
+    border-bottom: $border-1;
+    padding: 1rem 0;
+  }
+
+  .area-4 {
+    grid-column: 1;
+    grid-row: 4;
+    border-bottom: $border-1;
+    padding: 1rem 0;
+  }
+
+  .area-5 {
+    grid-column: 1;
+    grid-row: 5;
+    padding: 1rem 0;
+  }
```

Comment:
Nitpick: this could be a little DRYer is a common ```.area``` class was used as well (given that they share the same ```grid-column``` and ```padding``` values. But that may lead to confusion later when the classes deviate, so I'm also fine with leaving as is.

Reply:
Going to leave as-is.  Not sure how and if styling may change when we make this a shared component.